src/hmem_ipc_cache.c

```diff
@@ -38,9 +38,9 @@ static int ipc_cache_add_region(struct ofi_mr_cache *cache, struct ofi_mr_entry
 {
 	int ret;
 
-	ret = ofi_hmem_open_handle(entry->info.iface, (void **)&entry->info.ipc_handle,
+	ret = ofi_hmem_open_handle(entry->info.iface, (void **)&entry->info.handle,
 				   entry->info.iov.iov_len, entry->info.device,
-				   &entry->info.ipc_mapped_addr);
+				   &entry->info.mapped_addr);
```

Comment:
nit: There are a couple of renames in this patch which would be easier to deal with as a separate patch.